patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -1,8 +1,24 @@
 diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
-index 036d03e36a13b7947f0a3cc1dfdc81823b080645..ab3eaf80f7f711d901d76cfb445afcec73c5ed51 100644
+index 036d03e36a13b7947f0a3cc1dfdc81823b080645..e1da33dd35fcbcb70747aab9ec743606574a880c 100644
 --- a/third_party/widevine/cdm/widevine.gni
 +++ b/third_party/widevine/cdm/widevine.gni
-@@ -42,6 +42,8 @@ enable_widevine_cdm_component =
+@@ -4,14 +4,7 @@
+ 
+ import("//build/config/chrome_build.gni")
+ import("//media/media_options.gni")
+-
+-declare_args() {
+-  # Enables Widevine key system support. Enabled by default in Google Chrome,
+-  # on Android and Fuchsia platforms.
+-  # Can be optionally enabled in Chromium on non-Android platforms. Please see
+-  # //src/third_party/widevine/LICENSE file for details.
+-  enable_widevine = is_chrome_branded || is_android || is_fuchsia
```

Comment:
**//brave/browser/sources.gni** needs to use **enable_widevine** guard. But, a below import error happened when importing this file in **//brave/browser/sources.gni**.

**//build/config/chrome_build.gni** -> //brave/build/config/brave_build.gni -> /brave/browser/sources.gni -> //third_party/widevine/cdm/widevine.gni -> **//build/config/chrome_build.gni**

To avoid this import error, I make a try to move the declaration of enable_widevine to //brave/build/features.gni. But, I'm not sure if reviewers think this makes sense. Could you let me know if you have better solution to fix the import error?

Reply:
This is definitely not ok to patch like, why would you remove the entire block plus empty spaces? Even if we were going to patch something here, this is most certainly not the minimal patch to accomplish that, but we're just going to keep running into these circular dependencies so we need to find a different way to handle this